oc-chef-pedant/spec/api/policies/complete_endpoint_spec.rb

```diff
@@ -804,7 +804,7 @@ def rm(assoc)
             end
           end
 
-          context "when the name is close to the maximum size" do
+          context "when the name is close to the maximum size", :chef_zero_quirks do
 
             # On Chef Zero, some backends will append `.json` to a file name,
             # which can exceed the common limit of 255 characters.
```

Comment:
FYI, you added both tests (250 and 255), so we're still testing 255 on server--this lets us keep a 250-char test on zero.